app/services/metadata_cloud_service.rb

```diff
@@ -63,6 +64,7 @@ def find_source_ids_for(oid)
       aspace_uri: aspace_uri,
       holding: holding,
       item: item,
+      visibility: visibility,
```

Comment:
I'd lean toward doing the mapping right here since it's short instead of the more c like style w/ the extra variable.